src/Subsampling/benchmark/choose_n_farthest_points.cpp

```diff
@@ -76,7 +76,16 @@ int main(int argc, char**argv) {
             << " vs metric "              << std::chrono::duration_cast<std::chrono::milliseconds>((time_stop2 - time_start2)).count()
             << "  (Boost version " << BOOST_VERSION << ")\n";
   if (dists != dists2 || results != results2) {
+    // Note that with many points, it often happens that 2 points with the same distance are swapped in the output.
     std::cerr << "Results differ\n";
```

Comment:
Does it happen that results differ ? I was not able to reach this code.

Reply:
With 100k points and seed 0, I get 1 swap. With 50k points, I need to try several seeds before I get a swap.